pkg/commands/git_commands/commit_loader.go

```diff
@@ -650,7 +650,10 @@ func (self *CommitLoader) getFirstPushedCommit(refName string) (string, error) {
 
 // getLog gets the git log.
 func (self *CommitLoader) getLogCmd(opts GetCommitsOptions) oscommands.ICmdObj {
-	config := self.UserConfig.Git.Log
+	logOrder := self.AppState.GitLogOrder
+	if logOrder == "" {
+		logOrder = self.UserConfig.Git.Log.Order
+	}
```

Comment:
This is the wrong place to do this, we should do this right after loading the app state. I pushed a fixup to do that.